src/common/utils/api-id.config.ts

```
@@ -28,6 +28,7 @@ export const APIID = {
   USER_RESET_PASSWORD_LINK: 'api.user.sendLinkForResetPassword',
   USER_FORGOT_PASSWORD: 'api.user.forgotPassword',
   USER_DELETE: 'api.user.delete',
+  USER_BULK_IMPORT: 'api.user.bulkImport',
```

ESLint

[error] Replace `'api.user.bulkImport'` with `"api.user.bulkImport"`

(prettier/prettier)